File: src/test/OLD_TEST_MATLAB/covariance.m
%covariance matrix for tests on synthetic data: A(i,j) = a^|i-j|
function[A] = covariance(p,a)

	A = a*ones(p,p);
	for i=1:p
		A(i,:) = A(i,:) .^ abs(i-(1:p));
	end

end

File: src/test/generate_test_data/R_test_helpers/covariance.R
covariance= function(p,a){
  A = matrix(a, p,p)
  for{i in 1:p}{
    for{k  in 1:p}{
      A[i,]= A[i,]^abs(i-k) 
    }
  }
  return(A=A)
}